deploy/configs/packages.yaml

```
@@ -72,6 +72,12 @@ packages:
       prefix: /usr
   petsc:
     variants: ~fortran
+  py-tensorflow:
+    buildable: false
+    version: [2.4.0]
+    externals:
+    - spec: py-tensorflow cuda_arch=70 ^cuda@11.0.2
+      prefix: /gpfs/bbp.cscs.ch/apps/hpc/external/spack-packages/tensorflow/17-06-2021/tensorflow240_venv_py37_cuda11
```

Dates should be in the other order and adhere to ISO-8601.
```suggestion
      prefix: /gpfs/bbp.cscs.ch/apps/hpc/external/spack-packages/tensorflow/2021-06-17/tensorflow240_venv_py37_cuda11
```

---

I don't think I came across ISO=8601! 😄  I was batting for [RFC 5322 "Internet Message Format"](https://datatracker.ietf.org/doc/rfc5322/?include_text=1) ! 

We are getting rid of external strategy and moving to wheel!